File: C/First and last digit.c
//This code simply shows the first and last digit of the entered number, and the sum of both.
#include <stdio.h>

int main(){
    //Declaring variables...
    int number, first, last;
    number = first = last = 0;
    //Asking for input.
    scanf("%d", &number);
    //Operation to get the last number.
    last = number % 10;
    //Operation to get the first number.
    while ( number >= 10 ){
        number = number/10;
    }
    first = number;
    //Printing the results...
    printf("The last digit is: %d\n", last);
    printf("The first digit is: %d\n", first);
    //The number variable now adquires the sum of the first and last digits.
    number = first+last;
    printf("The sum of the first and last digits is: %d\n", number);
    return 0;
}

File: Python/First and Last digit.py
#This code simply asks for a number, and shows the first and last digit of that number. :)

#First we start off by asking the user a input.
n = int(input("Enter a number: "));
#We then set a "last" variable, to store the last number, by using the following operation.
last = n%10;
#While loop to get the first digit.
while (n >= 10):
    n = n/10;
#After that, we set the n variable to be an Integer.
#If that doesn't happen, the program will give the floating point, which isn't what we want in this case.
n = int(n);
#After that, the program will print the results.
print("The first digit is: " + str(n));
print("The last digit is: " + str(last));